### PR TITLE
Update graphic-card.md

### PR DESCRIPTION
**Pull request by Pitoniso**
Now yes, I hope

### CODE REVIEW

**Comment by Pitoniso** [comment]
Hi. I am surprise about the sentence on empirical testings and verified methods, when we do not have a strict pattern to follow due many of our simpthoms are triggered for differents reasons. I think we have to put on the wiki the informtion we have, not as a unquestionable truth, but as a  possibility. This may help, this could improve, or to some users it happens that. In my opinion, is to do this or to do nothing, and wiki is a usefull tool which is a very few profit is being taken from it

Is this only for the Nvidia youtube video?
Is this why the last 2 edits i did on the wiki didn't appear? The Windows and all of graphic cards?
________________________________
De: JTL <notifications@github.com>
Enviat el: dimecres, 22 d’abril de 2020 3:35:38
Per a: ledstrain/wiki.ledstrain.org <wiki.ledstrain.org@noreply.github.com>
A/c: Pitoniso <xatarra2@outlook.es>; Author <author@noreply.github.com>
Tema: Re: [ledstrain/wiki.ledstrain.org] Update graphic-card.md (#19)


@jtl999 commented on this pull request.

I appreciate your contributions to the wiki.

That being said I feel like we should only merge information that's been verified through empirical testing as opposed to tutorials from the masses on Nvidia forums trying to solve other (but possibly related) issues, and avoid leading others astray.

**Comment by jtl999** [comment]
> Hi. I am surprise about the sentence on empirical testings and verified methods, when we do not have a strict pattern to follow due many of our simpthoms are triggered for differents reasons. I think we have to put on the wiki the informtion we have, not as a unquestionable truth, but as a  possibility. This may help, this could improve, or to some users it happens that. In my opinion, is to do this or to do nothing, and wiki is a usefull tool which is a very few profit is being taken from it

I'm not questioning the use of the wiki to document information, that's what it's for, just that the wiki serves as a place to to more easily curate a set of information with some sort of validation associated with it as part of the (informal) quality standards we set as part of this project, and as a consequence will be more organized as opposed to a free-form discussions that can get off topic.

> Is this why the last 2 edits i did on the wiki didn't appear? The Windows and all of graphic cards?

Yes. Changes the Wiki have to be a approved by an authorized committer.

I've deferred to Slacor for his thoughts on this as well. Hopefully he'll see this and respond.

File: content/en/docs/hardware/graphic-card.md [hunk 1]
```diff
@@ -10,16 +10,21 @@ weight: 7
...
+The dithering parameter related in AMD cards is the one called saturation.
 
--DISABLE DITHERING IN INTEL CARDS:
+### Intel Cards
 
-There is no parameter to be disabled on Control panel to get rid of dithering on Intel cards, so we have to use 3rd party applications. In windows we can use "ditherig" and in Linux igfxtweak can be used, as for sure another ways to achieve it. Those 3rd party apps could work on Nvidia or AMD cars, as well
+Intel's included Control Panel does not currently allow configuration of dithering.  
+Some 3rd party applications are available that may allow dithering to be configured.  
+* Windows: `ditherig`
+* Linux: `igfxtweak`
```
Comment (jtl999) [inline content/en/docs/hardware/graphic-card.md:30]: igfxtweak isn't a Linux utility.

Comment (jtl999) [reply content/en/docs/hardware/graphic-card.md:30]: If I was going to merge any part of this pull request I'd atleast mention the `ditherig` utility for Intel graphics with a disclaimer saying "results may very". Too many uncertainties with the AMD and Nvidia sections.

Comment (tyler71) [reply content/en/docs/hardware/graphic-card.md:30]: Yeah, I was trying to rephrase it into something that would fit better.

**Comment by tyler71** [comment]
Have there been any posts or supporting information about changing these values with nvidia / amd / intel cards? I'm going to close the pull request for now, but I'll revisit it if we can identify some supporting evidence.

**Comment by jtl999** [comment]
> Have there been any posts or supporting information about changing these values with nvidia / amd / intel cards?

ditherig attempts to disable dithering through low level GPU control registers that should in theory bypass the driver. Other than that, no, probably just anecdotal testing.

**Comment by Pitoniso** [comment]
Aren't you really going to post this info? Do you understand how discouraging this is for a person who is trying to improve the wiki? Also I am probably the only person wanting to spend time on that wiki, so if you block my attempts to continue with this work the wiki will be useless
________________________________
De: JTL <notifications@github.com>
Enviat el: dissabte, 16 de maig de 2020 19:45
Per a: ledstrain/wiki.ledstrain.org <wiki.ledstrain.org@noreply.github.com>
A/c: Pitoniso <xatarra2@outlook.es>; Author <author@noreply.github.com>
Tema: Re: [ledstrain/wiki.ledstrain.org] Update graphic-card.md (#19)


Have there been any posts or supporting information about changing these values with nvidia / amd / intel cards?

ditherig attempts to disable dithering through low level GPU control registers that should in theory bypass the driver. Other than that, no, probably just anecdotal testing.

**Pull request closed without merging**